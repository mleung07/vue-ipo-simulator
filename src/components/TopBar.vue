<template>
  <div class="header">
    <div class="title">IPO Simulator</div>
  </div>
</template>

<script>
export default {
  name: "TopBar",
};
</script>

<style lang="scss" scooped>
.header {
  background-color: black;
  padding: 12px 16px;
  .title {
    font-size: 24px;
    color: white;
  }
}
</style>
