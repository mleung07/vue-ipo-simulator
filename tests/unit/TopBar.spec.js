import { shallowMount, mount } from "@vue/test-utils";
import TopBar from "@/components/TopBar.vue";

describe("TopBar.vue", () => {
  it("renders title", () => {
    const wrapper = shallowMount(TopBar);
    expect(wrapper.text()).toMatch("IPO Simulator");
  });

  it("matches snapshot", () => {
    const wrapper = mount(TopBar);
    expect(wrapper.element).toMatchSnapshot();
  });
});
