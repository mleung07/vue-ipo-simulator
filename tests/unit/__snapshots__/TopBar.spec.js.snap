// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar.vue matches snapshot 1`] = `
<div
  class="header"
>
  <div
    class="title"
  >
    IPO Simulator
  </div>
</div>
`;
